### PR TITLE
Added ethereum namespace api functions without implementation

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

This PR introduces adding empty ethereum api function declarations.
The logic should be implemented in here.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by aeharvlee — APPROVED** [review]
LGTM :) Thanks for your hard work 👍

**Review by aidan-kwon — DISMISSED** [review]

**Review by jeongkyun-oh — DISMISSED** [review]

File: api/api_ethereum.go [hunk 1]
```diff
@@ -70,3 +79,603 @@ func (api *EthereumAPI) SetPublicTransactionPoolAPI(publicTransactionPoolAPI *Pu
...
+}
+
+// Resend accepts an existing transaction and a new gas price and limit. It will remove
+// the given transaction from the pool and reinsert it with the new gas price and limit.
+func (api *EthereumAPI) Resend(ctx context.Context, sendArgs EthTransactionArgs, gasPrice *hexutil.Big, gasLimit *hexutil.Uint64) (common.Hash, error) {
+	// TODO-Klaytn: Not implemented yet.
+	return common.HexToHash("0x"), nil
+}
+
+// Accounts returns the collection of accounts this node manages
```
Comment (sirano11) [inline api/api_ethereum.go:677]:
```suggestion
// Accounts returns the collection of accounts this node manages.
```

**Review by aidan-kwon — DISMISSED** [review]

File: api/api_ethereum.go [hunk 2]
```diff
@@ -70,3 +79,603 @@ func (api *EthereumAPI) SetPublicTransactionPoolAPI(publicTransactionPoolAPI *Pu
 func (api *EthereumAPI) SetPublicAccountAPI(publicAccountAPI *PublicAccountAPI) {
 	api.publicAccountAPI = publicAccountAPI
 }
+
+// Etherbase is the address that mining rewards will be send to
```
Comment (sirano11) [inline api/api_ethereum.go:83]:
```suggestion
// Etherbase is the address that mining rewards will be send to.
```

File: api/api_ethereum.go [hunk 3]
```diff
@@ -70,3 +79,603 @@ func (api *EthereumAPI) SetPublicTransactionPoolAPI(publicTransactionPoolAPI *Pu
...
+func (api *EthereumAPI) Etherbase() (common.Address, error) {
+	// TODO-Klaytn: Not implemented yet.
+	return common.StringToAddress("0x0"), nil
+}
+
+// Coinbase is the address that mining rewards will be send to (alias for Etherbase)
```
Comment (sirano11) [inline api/api_ethereum.go:89]:
```suggestion
// Coinbase is the address that mining rewards will be send to (alias for Etherbase).
```

File: api/api_ethereum.go [hunk 4]
```diff
@@ -70,3 +79,603 @@ func (api *EthereumAPI) SetPublicTransactionPoolAPI(publicTransactionPoolAPI *Pu
...
+func (api *EthereumAPI) Coinbase() (common.Address, error) {
+	// TODO-Klaytn: Not implemented yet.
+	return common.StringToAddress("0x0"), nil
+}
+
+// Hashrate returns the POW hashrate
```
Comment (sirano11) [inline api/api_ethereum.go:95]:
```suggestion
// Hashrate returns the POW hashrate.
```

File: api/api_ethereum.go [hunk 5]
```diff
@@ -70,3 +79,603 @@ func (api *EthereumAPI) SetPublicTransactionPoolAPI(publicTransactionPoolAPI *Pu
...
+}
+
+// GetUncleByBlockHashAndIndex returns the uncle block for the given block hash and index. When fullTx is true
+// all transactions in the block are returned in full detail, otherwise only the transaction hash is returned.
+func (api *EthereumAPI) GetUncleByBlockHashAndIndex(ctx context.Context, blockHash common.Hash, index hexutil.Uint) (map[string]interface{}, error) {
+	// TODO-Klaytn: Not implemented yet.
+	return nil, nil
+}
+
+// GetUncleCountByBlockNumber returns number of uncles in the block for the given block number
```
Comment (sirano11) [inline api/api_ethereum.go:378]:
```suggestion
// GetUncleCountByBlockNumber returns number of uncles in the block for the given block number.
```

File: api/api_ethereum.go [hunk 6]
```diff
@@ -70,3 +79,603 @@ func (api *EthereumAPI) SetPublicTransactionPoolAPI(publicTransactionPoolAPI *Pu
...
+func (api *EthereumAPI) GetUncleCountByBlockNumber(ctx context.Context, blockNr rpc.BlockNumber) *hexutil.Uint {
+	// TODO-Klaytn: Not implemented yet.
+	return nil
+}
+
+// GetUncleCountByBlockHash returns number of uncles in the block for the given block hash
```
Comment (sirano11) [inline api/api_ethereum.go:384]:
```suggestion
// GetUncleCountByBlockHash returns number of uncles in the block for the given block hash.
```

File: api/api_ethereum.go [hunk 7]
```diff
@@ -70,3 +79,603 @@ func (api *EthereumAPI) SetPublicTransactionPoolAPI(publicTransactionPoolAPI *Pu
...
+	return nil
+}
+
+// GetRawTransactionByBlockHashAndIndex returns the bytes of the transaction for the given block hash and index.
+func (api *EthereumAPI) GetRawTransactionByBlockHashAndIndex(ctx context.Context, blockHash common.Hash, index hexutil.Uint) hexutil.Bytes {
+	// TODO-Klaytn: Not implemented yet.
+	return nil
+}
+
+// GetTransactionCount returns the number of transactions the given address has sent for the given block number
```
Comment (sirano11) [inline api/api_ethereum.go:529]:
```suggestion
// GetTransactionCount returns the number of transactions the given address has sent for the given block number.
```

File: api/api_ethereum.go [hunk 8]
```diff
@@ -70,3 +79,603 @@ func (api *EthereumAPI) SetPublicTransactionPoolAPI(publicTransactionPoolAPI *Pu
...
+func (api *EthereumAPI) GetTransactionCount(ctx context.Context, address common.Address, blockNrOrHash rpc.BlockNumberOrHash) (*hexutil.Uint64, error) {
+	// TODO-Klaytn: Not implemented yet.
+	return nil, nil
+}
+
+// GetTransactionByHash returns the transaction for the given hash
```
Comment (sirano11) [inline api/api_ethereum.go:535]:
```suggestion
// GetTransactionByHash returns the transaction for the given hash.
```

**Review by sirano11 — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Review by jeongkyun-oh — APPROVED** [review]

**Pull request merged**